benchmarks/holoscan_flow_benchmarking/README.md

```diff
@@ -62,7 +62,7 @@ These python dependencies include:
 #### Automatic Build with Benchmarking
 
 ```bash
-./run build <application_name> [options] --benchmark
+./holohub build <application_name> [options] --benchmark
```

Comment:
Thanks @wyli for enhancing the CLI! I just want to confirm that this is compatible with what we discussed with @tbirdso and @jjomier. While I prefer the current format with `--benchmark`, I want to ensure that it doesn’t conflict with our discussion of using `./holohub run <APP> benchmark`, which defines benchmark as a mode. Thanks